### PR TITLE
Get images and mount points from kickstart file

### PR DESCRIPTION
**Pull request by a-lyupa**
Instead of using hard-coded ones.

Signed-off-by: Anastasia Lyupa <a.lyupa@samsung.com>

### CODE REVIEW

**Review by ilya-palachev — APPROVED** [review]
Good patch!

**Review by ilya-palachev — APPROVED** [review]

File: combirepo/kickstart_parser.py [hunk 1]
```diff
@@ -48,6 +48,24 @@ def get_repository_names(self):
...
+    def get_images_mount_points(self):
+        """
+        Gets the names of images and corresponding mount points
+        that are mentioned in the kickstart file.
+        """
+        with open(self.path, "r") as kickstart_file:
+            images_dict = {}
+            for line in kickstart_file:
+                if line.startswith("part /"):
+                    image = re.findall(r"--label=(\S+)", line)
```
Comment (rayslava) [inline combirepo/kickstart_parser.py:60]: Is mountpoint part the only possible place of `--label` in .ks file?

Comment (a-lyupa) [reply combirepo/kickstart_parser.py:60]: As I see in kickstart docs, it is. All mountpoint lines start with "part". There is a search among theese lines only.

Comment (rayslava) [reply combirepo/kickstart_parser.py:60]: Okay, LGTM then

**Pull request merged**